components/tooltip/doc/index.en-US.md

```diff
@@ -46,6 +46,10 @@ The following APIs are shared by `nz-tooltip`, `nz-popconfirm`, `nz-popover`.
 | `hide` | Hide |
 | `updatePosition` | Update position |
 
+## Exclude body element's scroll event need to refresh the position of CDK
+
+In the usage of toolip(including popconfirm、popover), body element's scroll event will update the position of tooltip. It will never update the position of tooltip if the scroll event is happend in a custom element. You can add `cdkScrollable` directive to achieve the goal
+
```

Comment:
link to https://material.angular.io/cdk/scrolling/api
and `import {ScrollingModule} from '@angular/cdk/scrolling';` is needed
and please change the pull request title to docs, it is not a feat